mods/cnc/rules/structures.yaml

```diff
@@ -72,6 +72,7 @@ FACT:
 	WithBuildingPlacedAnimation:
 	Power:
 		Amount: 0
+	ProvidesCustomPrerequisite@buildingName:
```

Comment:
You don't need trait identifiers if there is only a single instance of a trait on an actor type.